packages/synapse-react-client/.storybook/storybook-vite.config.mts

```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite'
+import { vitestConfig } from 'vite-config'
+
+export default defineConfig(
+  vitestConfig,
```

Comment:
Remove custom storybook config and just use the same config we use for apps!

I think we had a custom config before because we adopted the Storybook + Vite integration when it was pretty nascent, and our app config didn't work in Storybook. That integration has since matured, and our shared config seems to work just fine now.